WORKSPACE

```diff
@@ -33,6 +33,32 @@ load("@os_info//:os_info.bzl", "is_linux", "is_windows")
 # bazel dependencies
 haskell_repositories()
 
+load("@io_tweag_rules_haskell//haskell:cabal.bzl", "stack_install")
+
+stack_install(
+    name = "stackage",
+    packages = [
```

Comment:
Is there any way to override a package with a Bazel target? E.g. inject a Bazel built version of, say, `streaming`.

Reply:
What you can do is override the version found in the snapshot. So in this sense `packages` is a merge between the `packages` and `extra-deps` field in the `stack.yaml`. This is implied in the rule documentation but the documentation could be more explicit.